docs/SoftwareCapabilities.md

```diff
@@ -0,0 +1,96 @@
+Static sealed objects in CHERIoT RTOS
+=====================================
+
+CHERIoT is a capability system.
+This extends from the hardware (all memory accesses require an authorising capability in a register, provided as an operand to an instruction) up through the software abstractions in the RTOS.
+In a capability system, any privileged operation requires an explicit capability to authorise it.
+
+Privilege is a somewhat fluid notion in a system with fine-grained mutual distrust but, in general, any operation that affects state beyond the current compartment is considered privileged.
+This includes actions such as:
+
+ - Acknowledging interrupts
+ - Allocating heap memory
+ - Establishing network connections
```

Comment:
More generally, external hardware is "state beyond the current compartment"... include "updating das blinkenlights" and "firing the missiles" in that list?